src/App.tsx

```diff
@@ -3,30 +3,60 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getData } from './api/api';
 
 interface State {
+  todos: Todo[];
   selectedUserId: number;
+  errorMessage: string;
 }
 
 class App extends React.Component<{}, State> {
   state: State = {
+    todos: [],
     selectedUserId: 0,
+    errorMessage: '',
+  };
+
+  async componentDidMount() {
+    try {
+      const todos = await getData('/todos');
```

Comment:
It'd be better to meke different methods for users and todos and not pass the  exact path